src/apiData/apiCalls.js

```diff
@@ -0,0 +1,18 @@
+const baseURL = 'https://rancid-tomatillos.herokuapp.com/api/v2/'
+const apiCalls = {
+  fetchApiData: async (query) => {
+    const response = await fetch(`${baseURL}${query}`)
+    return await apiCalls.checkForError(response)
+  },
+
+  checkForError: response => {
+    if (!response.ok) {
+      throw new Error('Network response was not ok')
+    } 
+    
+    return response.json()
+  }
+
+}
+
+export default apiCalls
```

Comment:
Nice! Love the async await. What was behind the decision to make this an object, rather than a function?